### DISCUSSION TITLE
Does sidebar score affect the order of lines?

### DISCUSSION THREAD

**Discussion by VendoAU** [Minestom/Minestom#798] [Help] [answered]
If I create a sidebar will the order of the lines be kept the same if all of the scores are zero, or do they need to be descending?

**Comment by wasn325** [comment]
I think the primary sorting is by the scores and the secondary by the name 

Like
§1Text
§2Text
§3Text

should work

**Comment by emortaldev** [comment, marked as answer]
The sorting is client side and it is ordered by highest at the top and lowest at bottom

I think if they're all 0 its just order of creation, give it a go!

**Reply by VendoAU** [reply]
That makes sense, thanks for your help.
